backend/corpora/common/corpora_orm.py

```diff
@@ -163,7 +163,7 @@ class DbProject(Base):
 
     # Relationships
     user = relationship("DbUser", uselist=False, back_populates="projects")
-    links = relationship("DbProjectLink", back_populates="projects")
+    links = relationship("DbProjectLink", back_populates="project")
```

Comment:
`project` or `projects`?

Reply:
Actually, the correction would be on line 165?

Reply:
The code as proposed seems right to me. Project:ProjectLink and User:Project are both 1:N

Reply:
I will keep the naming as is

Reply:
oh didn't realize that both `project` and `projects` are valid - thought it was a typo/inconsistent. Resolving